src/formatDate.js

```diff
@@ -8,7 +8,70 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [, , , slider] = fromFormat;
+  const [, , , glue] = toFormat;
```

Comment:
The code assumes that the fourth element in `fromFormat` and `toFormat` arrays is the delimiter. This assumption might not always hold true, especially if the format arrays do not have exactly four elements. Consider adding a check or a more flexible way to determine the delimiter.